tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -45,15 +45,21 @@ def test_dcv_configuration(
     # dcv connect show url
     env = operating_system.environ.copy()
     env["AWS_DEFAULT_REGION"] = region
-    # Disable StrictHostKeyChecking by adding option to end of file
-    operating_system.system(r"echo '    StrictHostKeyChecking no' >> /etc/ssh/ssh_config")
-    result = run_command(["pcluster", "dcv", "connect", cluster.name, "--show-url"], env=env)
+
+    # add ssh key to global known hosts file to avoid ssh keychecking prompt
+    host_keys_file = "/etc/ssh/ssh_known_hosts"
+    operating_system.system("ssh-keyscan -t rsa {0} > {1}".format(cluster.master_ip, host_keys_file))
```

Comment:
Could we extract also this part into utils as add_keys_to_known_hosts(host_name, key_file)?